wavelength-plugins/profile.js

```diff
@@ -275,6 +282,81 @@ exports.commands = {
 		},
 	},
 
+	bg: 'background',
+	background: {
+		set: 'setbg',
+		setbackground: 'setbg',
+		setbg: function (target, room, user) {
+			if (!this.can('broadcast')) return false;
```

Comment:
shouldnt this be lock?

Reply:
oh my bad LOL that's what we use on my server